File: tkk-java/src/tesKompetensiTeknis/NomorEmpat.java
package tesKompetensiTeknis;

import java.util.Scanner;

public class NomorEmpat {
    public static String reverseWords(String words)
    {
        String reversedWord = "";

        char[] charArray = words.toCharArray();

        for(int i=charArray.length-1; i>=0; i--){
            reversedWord += charArray[i];
        }

        return reversedWord;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Masukkan Kata/Kalimat : ");

        String words = scanner.nextLine();

        System.out.println(reverseWords(words));
    }
}

File: tkk-dart/bin/nomor_empat.dart
import 'dart:io';

String reverseWords(String words){
  var reversedWord = '';

  var charArray = words.split('');

  for(var i=charArray.length-1; i>=0; i--){
    reversedWord += charArray[i];
  }

  return reversedWord;
}

void main(List<String> arguments) {
  print('Masukkan Kata/Kalimat : ');
  var words = stdin.readLineSync().toString();

  print(reverseWords(words));
}
